src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,63 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * This class is  for the specific implementation
```

Comment:
@sebing please describe that "specific" implementation more, saying that it's just specific is not enough